src/main/kotlin/no/nav/syfo/util/VarselUtil.kt

```diff
@@ -25,6 +27,22 @@ class VarselUtil(private val databaseAccess: DatabaseInterface) {
             .filter { it.type == varselType.name }
     }
 
+    fun isVarselInEffect(fnr: String, varselType: VarselType): Boolean {
```

Comment:
kan denne metoden få et annet navn? Jeg synes i hvert fall at det er litt vanskelig å forstå "InEffect". Hva med "isVarselSendt" f.eks.?

Reply:
@kaja-nav Det er en sjekk på hvorvidt nytt varsel kan sendes (om utsendt varsel er gjeldende / "in effect", uke 39-52), ikke om hvorvidt varsel er sendt (som kan skje flere ganger over flere sykeforløp), men kan godt rename den 👍

Reply:
Fikset 👍